pep/app/pods/read/document/controller.ts

```diff
@@ -133,6 +142,26 @@ export default class ReadDocument extends Controller {
             : this.paginator.models;
     }
 
+    /**
+     * Transform the sorting to a format the API can handle
+     *
+     * @param {string[]} sorts
+     * @returns
+     * @memberof Search
+     */
+    @action
+    async onChangeSorting(sorts: string[]) {
+        if (sorts.length) {
+            return transformSearchSortToAPI(sorts);
+        } else {
+            return [];
+        }
+    }
```

Comment:
unnecessary `async`

could we move the 
```ts
if(!sorts.length) {
    return [];
}
```
logic into the `transformSearchSortToAPI()` util function, turning this into just
```ts
@action
onChangeSorting(sorts: string[]) {
    return transformSearchSortToAPI(sorts);
}
```

Reply:
IMO I don't think we should be calling that function if we don't have any sorts. What about

```
return sorts.length ? transformSearchSortToAPI(sorts) : [];
```

Reply:
if you are doing this same guard in multiple places where thats used i think its worthwhile to have the logic inside it, to just make it more robust it how it handles the args its given, its not like you are breaking its type contract, you still are passing it `string[]`.

that being said, i'm fine with your proposed change above.